custom/inddex/fixtures.py

```diff
@@ -113,6 +121,14 @@ def foods(self):
     def foods_by_name(self):
         return {food.food_name: food for food in self.foods.values()}
 
+    @cached_property
+    def _nutrient_names_by_code(self):
```

Comment:
That's true.  I don't think I'm going to need the `nutrient_name` at all, so I went this way for brevity.  Perhaps I should drop the other fields before constructing the `Nutrient` instances.

> I also noticed that "name_unit" is very similar to a slug, except that it satisfies the requirements of a Python variable name (names that would start with a digit are prefixed with "_", and slugs use "_" instead of "-"). Any reason for that?

Not sure - as I understand it, these fixtures are generated by a third party web app.  I suspect that is their implementation leaking into the interface.